Sources/xcproj/PBXTarget.swift

```diff
@@ -88,7 +88,7 @@ public class PBXTarget: PBXObject {
     }
 
     func plistValues(proj: PBXProj, isa: String, reference: String) -> (key: CommentedString, value: PlistValue) {
-        var dictionary: [CommentedString: PlistValue] = [:]
+        var dictionary = plistValues(proj: proj, reference: reference)
```

Comment:
Shouldn't this be `super.plistValues`?

Reply:
Yeah, I can update that to make it more clear. This method has an `isa:` named argument so this code isn't infinitely recursive. After reading your comment I was like, wait, how *does* this work? :)